homeassistant/components/sensor/wunderground.py

```diff
@@ -154,18 +173,23 @@ def device_state_attributes(self):
 
         attrs[ATTR_ATTRIBUTION] = CONF_ATTRIBUTION
 
-        if not self.rest.alerts or self._condition != 'alerts':
+        if self._condition != 'alerts':
             return attrs
 
-        multiple_alerts = len(self.rest.alerts) > 1
-        for data in self.rest.alerts:
-            for alert in ALERTS_ATTRS:
-                if data[alert]:
-                    if multiple_alerts:
-                        dkey = alert.capitalize() + '_' + data['type']
-                    else:
-                        dkey = alert.capitalize()
-                    attrs[dkey] = data[alert]
+        if self._condition == 'alerts':
```

Comment:
There is no need for this added `if`-statement. The condition above will ensure that `_condition` is equal to `alerts`.